### PR TITLE
Add .coafile

### PR DESCRIPTION
**Pull request by rag594**
As of now only SpaceConsistencyBear is used for checking usage of spaces instead of tabs, please
review this @rhemon. After your approval I will add more checks

### CODE REVIEW

File: .coafile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[Spacing]
+files = ./**/*.py
```
Comment (rhemon) [inline .coafile]: it should have been fine if you just used **.py, right?

File: rss-generator.py.orig [hunk 2]
```diff
@@ -0,0 +1,312 @@
+DESC = """
```
Comment (rhemon) [inline rss-generator.py.orig]: since we are ignoring this, shouldn't this be removed?

File: test.py.orig [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import unittest
```
Comment (rhemon) [inline test.py.orig]: same here, shouldn't this be removed?

**Comment by rhemon** [comment]
if you remove the `.orig` files, please make sure to ammend, and not have a separate commit.
other changes LGTM.
you can move on and add other bears 👍

File: .coafile (hunk 1 above)
Comment (rag594) [inline .coafile]: Yes, I'll change that in next commit

**Comment by rag594** [comment]
Sorry for the latest commit , in this I finally removed the `.orig` files. After reviewing you can merge with master branch.I'll add more checks soon.

File: .coafile [hunk 4]
```diff
@@ -0,0 +1,5 @@
+[Spacing]
```
Comment (rhemon) [inline .coafile]: I think there should be two different section, one [Default] that will just have SpaceConsistencyBear and a [Python] section that will have the other two bears.

Comment (rag594) [inline .coafile]: That's correct or we could have different sections for each bear?

**Comment by rhemon** [comment]
and sorry I don't have access to merge, so you have to wait until someone with access has a look. 
also you might want to squash all of them into 1 commit, although I am not sure if this project recommends that. its not bad to have separate reasonable commits. which can be judged differently. e.g. according to me. having commits for each bear is reasonable, while also having 1 whole commit for adding coala is also reasonable

File: .coafile (hunk 4 above)
Comment (rhemon) [inline .coafile]: no I dont think there is any need to have different section for each bear. 
distributing sections based on language should be good enough.

**Comment by rhemon** [comment]
just so you know, its not necessary you have to have new commit every time, you can simply use `git commit --amend` which updates your previous commit.

**Comment by rag594** [comment]
@mariobehling kindly review this pull request whenever you are free.

**Pull request merged**